### PR TITLE
ARM64 feature changelog

### PR DESCRIPTION
**Pull request by olmisa**
* [x] Has CHANGELOG.md been updated?
 * [ ] Are tests passing locally?
 * [ ] Are the files formatted correctly?
 * [ ] Did you add unit tests if this modifies the Windows code?
 * [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Changelog updated.

## Misc

[AB#74488](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/74488)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Version 2.6.5 （Under development）
 
+### App Center
+
+* **[Feature]** Support ARM64 architecture.
```
Comment (MatkovIvan) [inline CHANGELOG.md]: We must mention that minimal versions have been changed

Comment (olmisa) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 ## Version 2.6.5 （Under development）
 
+### App Center
+
+* **[Feature]** Support ARM64 architecture. .Net Framework updated to 4.6.1. Microsoft.NETCore.UniversalWindowsPlatform updated to 6.2.8. UWP min version updated to 10.0.16299.0.
```
Comment (MatkovIvan) [inline CHANGELOG.md]:
```suggestion
* **[Feature]** Support ARM64 architecture for UWP platform.
* **[Breaking change]** Bump minimal versions: .NET Framework to 4.6.1 for WPF projects and minimal OS version to 16299 for UWP projects.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
 
 ## Version 2.6.5 （Under development）
 
+### App Center
+
+* **[Feature]** Support ARM64 architecture for UWP platform.
+* **[Breaking change]** Bump minimal versions: .NET Framework to 4.6.1 for WPF projects and minimal OS version to 16299 for UWP projects.
```
Comment (guperrot) [inline CHANGELOG.md]:
```suggestion
* **[Breaking change]** Bump minimal versions: .NET Framework to 4.6.1 for WPF/WinForms projects and minimal OS version to 16299 for UWP projects.
```

**Pull request merged**